minian/utilities.py

```diff
@@ -825,7 +825,7 @@ def get_optimal_chk(ref, arr=None, dim_grp=None, ncores='auto', mem_limit='auto'
     if ncores=='auto':
         ncores = psutil.cpu_count()
     if mem_limit=='auto':
-        mem_limit = (psutil.virtual_memory().available / (1024 ** 2))
+        memlimit = (psutil.virtual_memory().available + psutil.swap_memory().free) / (1024 ** 2)
```

Comment:
looks like the variable name changed. Did you tested this? Are there other changes I'm missing?

Reply:
ah yes sorry didn't push it... done